### PR TITLE
Trigger metrics and ui build post uninstall

### PR DESCRIPTION
**Pull request by rajeevr2715**

### CODE REVIEW

File: feature-extensions/apigee/setup.py [hunk 1]
```diff
@@ -148,7 +148,11 @@ def install(args):
     # Store the CREDENTIAL_ID in jenkins
     setCredential(args, credential_id)
     # Trigger metrics api
-    startJob(args)
+    metricJobUrl = "job/build-pack-api/buildWithParameters?token=jazz-101-job&service_name=metrics&domain" \
+                   "=jazz&scm_branch=master"
+    startJob(args, metricJobUrl)
+    # Trigger jazz ui
+    startJob(args, "job/jazz_ui/build")
```
Comment (SiniWilson) [inline feature-extensions/apigee/setup.py]: startJob(args, "job/jazz_ui/build?token=jazz-101-job")

File: feature-extensions/apigee/setup.py [hunk 2]
```diff
@@ -173,6 +177,12 @@ def uninstall(args):
         args.scm_password,
         args.scm_pathext
     )
+    # Trigger metrics api
+    metricJobUrl = "job/build-pack-api/buildWithParameters?token=jazz-101-job&service_name=metrics&domain" \
+                   "=jazz&scm_branch=master"
+    startJob(args, metricJobUrl)
+    # Trigger jazz ui
+    startJob(args, "job/jazz_ui/build")
```
Comment (SiniWilson) [inline feature-extensions/apigee/setup.py]: startJob(args, "job/jazz_ui/build?token=jazz-101-job")

Comment (rajeevr2715) [reply feature-extensions/apigee/setup.py]: Done.

File: feature-extensions/apigee/setup.py (hunk 1 above)
Comment (rajeevr2715) [reply feature-extensions/apigee/setup.py]: Done.

**Pull request merged**